src/hooks/sales.ts

```diff
@@ -293,18 +283,30 @@ export const useManagerLeads = (manager: Manager) => {
 
     if (authToken && manager && manager.id && appId) {
       try {
-        const payload = {
-          managerId: manager.id,
-          appId: appId,
-        }
-
-        const { data } = await axios.post(
-          `${process.env.REACT_APP_LODESTAR_SERVER_ENDPOINT}/members/saleLeadMemberData`,
-          payload,
-          { headers: { authorization: `Bearer ${authToken}` } },
-        )
-
-        setSalesLeadMemberInfo(data)
+        await axios
+          .post(
+            `${process.env.REACT_APP_KOLABLE_SERVER_ENDPOINT}/kolable/sales-lead/${manager.id}`,
+            {},
+            {
+              headers: { authorization: `Bearer ${authToken}` },
+            },
+          )
+          .then(({ data }) => {
+            const result = data.map((salesLeadMember: SalesLeadMember) => ({
+              ...salesLeadMember,
```

Comment:
It's good to explicitly list each field with conversion when the results come from an external api